code/modules/atmospherics/machinery/components/unary_devices/machine_connector.dm

```diff
@@ -106,4 +107,23 @@
 	if(node)
 		node.atmos_init()
 		node.add_member(gas_connector)
+		gas_connector.update_parents()
 	SSair.add_to_rebuild_queue(gas_connector)
+
+/**
+ * Handles air relocation to the pipe network/environment
+ */
+/datum/gas_machine_connector/proc/relocate_airs(mob/user)
+	var/turf/T = get_turf(connected_machine)
+	var/datum/gas_mixture/inside_air = gas_connector.airs[1]
+	if(inside_air.total_moles() > 0)
+		if(gas_connector.nodes[1])
```

Comment:
make this an early continue, i.e.

```suggestion
		if(!gas_connector.nodes[1]) // or maybe isnull() if this is a typed variable
			T.assume_air(inside_air) // this variable name should still be changed
			return
			
		// rest of the code that was previously in the if() block is now here
```

this is just nice for readability since everything won't be indented